controlled-vocabulary/src/main/java/org/uniprot/core/cv/xdb/UniProtXDbTypeDetail.java

```diff
@@ -6,16 +6,18 @@
 import java.util.Objects;
 
 public class UniProtXDbTypeDetail implements Serializable {
+
     private static final long serialVersionUID = 8751881513996820892L;
+    public static String EXPLICIT = "Explicit";
     private static final DBXRefTypeAttribute DEFAULT_ATTRIBUTE =
             new DBXRefTypeAttribute("Description", "description", null);
     private String name;
     private String displayName;
     private DatabaseCategory category;
     private String uriLink;
     private List<DBXRefTypeAttribute> attributes;
-    private String linkTp ="Explicit";
-    private String linkedReason= null;
+    private String linkTp = EXPLICIT;
```

Comment:
what is `linkTp`? If it's link type, should we call it `linkType` to be clearer?